### ISSUE TITLE
underscore in tag name turns up no results

### ISSUE THREAD

**Issue by derekvan** [Aidurber/tag-page-preview#3]
Thanks for this useful plugin. Works great with most tags, but some of my tags have an underscore for separate words, like "#test_tag" With these tags, the plugin produces an error: 

"Pages with #"

In other words, it looks like an underscore creates an issue where the search term for the tag is blank.

**Comment by Aidurber** [comment]
Interesting, I'll take a look!

**Comment by Aidurber** [comment]
Posting for my benefit.
Last week, I noticed an issue clicking on tags with more than 1 level, the parts of the tag were all split into separate `span` elements. I updated the edit mode code so it would find all of the tags that matched such as `cm-tag-abc` or `cm-tag-dndhomebrew` this was sufficient to find all parts of the tag. They (Obsidian or CodeMirror) seem to have patched that a bit better so subtags all go into the same `span`. 

I think there's a bug with Obisidian/CodeMirror in that any tag with an underscore gets placed in another span **but** the `cm-tag-` does not contain the correct tag. It's treated as a separate tag. 

![CleanShot 2021-09-06 at 16 23 48@2x](https://user-images.githubusercontent.com/5732291/132238821-10cee447-bb33-479f-8e51-04ec581a7b1c.jpg)

I need to find a nice way to walk previous siblings and next of the click target until I have a full range of nodes that resemble the clicked tag. It's not quite trivial. I'll keep playing with it until I can find a robust selector.
